data-reconciliation-app/src/app.json

```diff
@@ -1,21 +1,53 @@
 {
   "endpoints": {
-    "/votes": {
-      "get": {
-        "js_module": "endpoints/sample.js",
-        "js_function": "read",
-        "forwarding_required": "sometimes",
-        "authn_policies": [],
-        "mode": "readonly",
-        "openapi": {}
-      },
+    "/ingest": {
       "post": {
-        "js_module": "endpoints/sample.js",
-        "js_function": "write",
-        "forwarding_required": "sometimes",
-        "authn_policies": [],
+        "js_module": "endpoints/ingest.js",
+        "js_function": "submitData",
+        "forwarding_required": "always",
+        "authn_policies": ["member_cert"],
         "mode": "readwrite",
```

Comment:
This endpoint only writes?

Reply:
It will perform read and update operations to key-value store